DQMOffline/RecoB/interface/JetTagPlotter.h

```diff
@@ -80,6 +84,12 @@ class JetTagPlotter : public BaseBTagPlotter {
 
   // reconstructed jet phi
   FlavourHistograms<double> * dJetRecPhi;
+  
+  // jet Phi larger than requested discrimnator cut
+  FlavourHistograms<double> * dJetPhiDiscrCut;
```

Comment:
Btw, I have a perhaps naive question about that: since `DQMStore::IBooker::book*()` and `DQMStore::IGetter::get()` return raw `MonitorElement*`, pointers to these objets used in our classes cannot be moved to `shared_ptr`, right?